Controller/Process/Json.php

```diff
@@ -432,4 +450,26 @@ protected function _returnResult($message)
             ->setBody($message);
         return;
     }
+
+    /**
+     * Returns true when the event code support HMAC validation
+     *
+     * @param $response
+     */
+    protected function _isHmacSupportedEventCode($response)
```

Comment:
please remove the underscore we don't use this anymore as described in the ruleset of PSR2